### DISCUSSION TITLE
What happened to Openpose-hands?

### DISCUSSION THREAD

**Discussion by mylife4aiur5** [Mikubill/sd-webui-controlnet#424] [General]
Openpose-hands is no longer available.

**Comment by missionfloyd** [comment]
[Disabled](https://github.com/Mikubill/sd-webui-controlnet/issues/210), Uncomment [this line](https://github.com/Mikubill/sd-webui-controlnet/blob/2ce17c0a2bb661ca186764c3b5fe79997cb267e6/scripts/controlnet.py#L180) to enable it.

**Reply by mylife4aiur5** [reply]
How can I find that line from Mac?

**Reply by missionfloyd** [reply]
Open `stable-diffusion-webui/extensions/sd-webui-controlnet/scripts/controlnet.py` in a text editor.
